docs/Tutorials/Private-Network/Create-Private-Clique-Network.md

```diff
@@ -124,16 +124,15 @@ In `extraData`, replace `<Node 1 Address>` with the [address for Node-1](#2-get-
 Start Node-1:
 
 ```bash tab="MacOS"
-besu --data-path=data --genesis-file=../cliqueGenesis.json --bootnodes --network-id 123 --rpc-http-enabled --rpc-http-api=ETH,NET,CLIQUE --host-whitelist="*" --rpc-http-cors-origins="all"      
+besu --data-path=data --genesis-file=../cliqueGenesis.json --network-id 123 --rpc-http-enabled --rpc-http-api=ETH,NET,CLIQUE --host-whitelist="*" --rpc-http-cors-origins="all"      
 ```
 
 ```bash tab="Windows"
-besu --data-path=data --genesis-file=..\cliqueGenesis.json --bootnodes --network-id 123 --rpc-http-enabled --rpc-http-api=ETH,NET,CLIQUE --host-whitelist="*" --rpc-http-cors-origins="all"    
+besu --data-path=data --genesis-file=..\cliqueGenesis.json --network-id 123 --rpc-http-enabled --rpc-http-api=ETH,NET,CLIQUE --host-whitelist="*" --rpc-http-cors-origins="all"    
 ```
 
 The command line specifies: 
 
-* No arguments for the [`--bootnodes`](../../Reference/CLI/CLI-Syntax.md#bootnodes) option because this is your bootnode
```

Comment:
Id leave this in saying its using the default config value of [ ] implying it is a bootnode?

Reply:
But why do that? Does it make a difference?

Reply:
@joshuafernandes - I removed this because specifying `--bootnodes` and leaving the bootnodes option completely have the same effect (empty list of bootnodes) and users were getting confused thinking a bootnode was different to a regular node (this is the case for Geth).  In cases with more than one bootnode, a user then thought we didn't have any bootnodes because there wasn't a node with only `--bootnodes` 

Does that make sense?  Let me know if you want to discuss.

Reply:
All good :) , whatever makes it easier for people to follow and use